lib/tapioca/dsl/pipeline.rb

```diff
@@ -68,6 +68,10 @@ def run(&blk)
           ERROR
         end
 
+        if defined?(::ActiveRecord::Base) && constants_to_process.any? { |c| c < ::ActiveRecord::Base }
```

Comment:
Small comment here: We've learned the hard way that it is not safe to call methods like `<` on arbitrary constants since they might have overloaded the method (here's looking at you `XPath`). Can we please rewrite this so that the call is being made on `ActiveRecord::Base` instead?
```suggestion
        if defined?(::ActiveRecord::Base) && constants_to_process.any? { |c| ::ActiveRecord::Base > c  }
```